translation/cr105ph1nx_README_fr.md

```diff
@@ -0,0 +1,125 @@
+## OpenMindsClub Joins Hacktoberfest 2020! ##
+
+![Event logo](https://www.pixenli.com/image/sbWyjrII)
+
+![GitHub Hacktoberfest combined status](https://img.shields.io/github/hacktoberfest/2020/openmindsclub/H4ckT0b3rF3st-2k20?color=FF1493&label=Openfest%202020)
+
+## Table des Matières:
+* [Qu'est-ce que le Hacktoberfest?](#Qu-est-ce-Que-Le-Hacktoberfest)
+* [Que Puis-Je Gagner?](#Que-Puis-Je-Gagner)
+* [Comment Effectuer Sa Première Contribution?](#Comment-Effectuer-Sa-Première-Contribution)
+* [Comment Réussir Le Défi Hacktoberfest?](#Comment-Réussir-Le-Défi-Hacktoberfest)
+* [Getting Started - Basic](#Getting-Started-Basic)
+* [Getting Started - Advanced](#Getting-Started-Advanced)
+* [Tâches et Défis](#Tâches-et-Défis)
+
+[![GitHub contributors](https://img.shields.io/github/contributors/openmindsclub/H4ckT0b3rF3st-2k20)](https://GitHub.com/Naereen/StrapDown.js/graphs/contributors/)
+[![GitHub pull-requests closed](https://img.shields.io/github/issues-pr-closed/openmindsclub/H4ckT0b3rF3st-2k20)](https://GitHub.com/Naereen/StrapDown.js/pull/)
+[![GitHub pull-requests](https://img.shields.io/github/issues-pr/openmindsclub/H4ckT0b3rF3st-2k20)](https://GitHub.com/Naereen/StrapDown.js/pull/)
+
+[![GitHub watchers](https://img.shields.io/github/watchers/openmindsclub/H4ckT0b3rF3st-2k20?style=social)](https://github.com/openmindsclub/H4ckT0b3rF3st-2k20/network)
+[![GitHub forks](https://img.shields.io/github/forks/openmindsclub/H4ckT0b3rF3st-2k20?style=social)](https://github.com/openmindsclub/H4ckT0b3rF3st-2k20/network)
+[![GitHub stars](https://img.shields.io/github/stars/openmindsclub/H4ckT0b3rF3st-2k20?style=social)](https://github.com/openmindsclub/H4ckT0b3rF3st-2k20/network)
+
+
+On espére que cet événement sera votre premier pas vers votre riche parcours de contribution aux projets Libre Open Source! <br/> <br/>
+En hostant Hacktoberfest 2020, OpenMinds Club vise à encourager tout le monde à contribuer aux projets Libre Open Source comme cela a été et restera toujours notre philosophie et notre objectif fondamental. <br/> Ce repository est ouvert à toute personne souhaitant commencer leur aventure de contribution. Quant aux débutants, vous êtes également les bienvenus à participer à notre Hacktoberfest! Assurez-vous d'en savoir plus sur l'[open source] (https://github.com/open-source). <br/> <br/>
```

Comment:
"organisant" instead of "hostant"

Reply:
Sure, my frenglish gets outta control sometimes